c/validation/adbc_validation.cc

```diff
@@ -32,6 +32,7 @@
 #include <gtest/gtest-matchers.h>
 #include <gtest/gtest.h>
 #include <nanoarrow/nanoarrow.h>
+#include <algorithm>
```

Comment:
nit, but sort this with the other stdlib headers

Reply:
i was relying on clang-format to sort the headers.... but i guess it didn't do it because I had put it in this block. I'll fix this.